apps/angular/src/core/utils/anime-managers-form-validate.utils.ts

```
@@ -0,0 +1,25 @@
+import { ValidatorFn, AbstractControl, ValidationErrors } from '@angular/forms';
+
+/**
+ * Checks if end date is after start date.
+ * @param startDateControlName Start date control name.
+ * @param endDateControlName End date control name.
+ */
+export function startEndDatesIntervalValidator(startDateControlName: string, endDateControlName: string): ValidatorFn {
+	return (formGroup: AbstractControl): ValidationErrors | null => {
+		if (formGroup.get(startDateControlName) === null && formGroup.get(endDateControlName) === null) {
+			return null;
```

I though we discussed that we should throw an error in this case. We want developers know that the validator doesn't work because they provided incorrect control names